### PR TITLE
 Set metrics prefix from a system property. 

### PR DESCRIPTION
**Pull request by pmelkowski**
New command line option `-Dknotx.metrics.options.prefix` to control the prefix.
+ The example configuration updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,39 +21,32 @@ Do the following steps to start sending Knot.x application metrics to Graphite:
...
-        "type": "file",
-        "format": "conf",
-        "config": {
-          "path": "${KNOTX_HOME}/conf/dashboardStack.conf"
-        }
-      }
-    ]
-...
+* Add `metricSender` entry to `application.conf` modules configuration
+```
```
Comment (malaskowski) [inline README.md]: Why no longer use metrics as a separate store?
Is there something wrong with this approach?

Comment (pmelkowski) [reply README.md]: I thought we no longer modified bootstrap.json, but included configuration files in application.conf. Is that correct?

Comment (malaskowski) [reply README.md]: It is more clear to me to define metrics as separate store entry.
Metrics does not introduce any business logic into Knot.x instance.

Comment (pmelkowski) [reply README.md]: Ok, reverted this change.

**Review by malaskowski — APPROVED** [review]
@pmelkowski thanks!

**Pull request merged**